patroni/ha.py

```diff
@@ -128,6 +128,26 @@ def is_leader(self):
         with self._is_leader_lock:
             return self._is_leader > time.time()
 
+    def is_static_primary(self):
+        """Check if this node is configured as the static primary of the cluster."""
+        if self.cluster and self.cluster.config and self.cluster.config.modify_index:
+            config = self.cluster.config.data
+        else:
+            config = self.patroni.config.dynamic_configuration
+        static_primary = config.get('static_primary')
```

Comment:
I would suggest to extract this block into a method in order to avoid duplication with https://github.com/zalando/patroni/pull/2318/files#diff-b6a3a0ef64783a6d32feb908d74a3a009e9f25008b8ca5313c9c9747e3b74a63R145-R149

Reply:
Will do.

Reply:
Done.